profiles/templates/404.html

```diff
@@ -44,8 +44,8 @@
     <div>
     <h1 class="center">Error 404</h1>
     <h2 class="center">Page does not exist!</h2>
-    <img class="center" src="{{ url_for('static', filename='/assets/images/tvstatic.gif') }}">
+    <img class="center" src="{{ url_for('static', filename='assets/images/tvstatic.gif') }}">
     <h2 id="message" class="center">Message: "{{ message }}"</h2>
     </div>
 </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
Is this a whitespace change? It's not rendering on mobile, whatever it is.

Reply:
No idea, nothing changed on my end. Maybe git is confused because there is no ending newline.

Reply:
Jk looks like a removed whitespace, though I never touched it personally.